src/migration/issue_227_draft.txt

```diff
@@ -0,0 +1,10 @@
+Steps necessary:
+    Delete all files in this folder.
+    Drop database(? Not 100% sure what this entails. Does this require me to empty the postgres database or to simply delete it?)
```

Comment:
Use the PG cli, PG Admin, or the Postgres App for Mac and literally delete the `hangar` database and then recreate it